docs/editors-guide.md

```diff
@@ -102,6 +102,34 @@ Consider using `data-transform="updateExample"` with `<!-- .. -->` comments to a
   </pre>
 ```
 
+### PREFIX/BASE use in examples
+
+Turtle and TriG examples should prefer the use of `PREFIX` and `BASE` to `@prefix` and `@base`. This creates a uniform style between Turtle/TriG and SPARQL and facilitates copy/paste.
```

Comment:
Turtle and SPARQL already support both forms (i.e., anyone can use any of `PREFIX`, `BASE`, `@prefix`, or `@base`); I think TriG does as well.

I was not aware that all of these forms were case insensitive (i.e., that `PREFIX`, `@PREFIX`, `@prefix`, `prefix`, and I presume any oddmix, like `PrEfIx`, are also valid). This should perhaps be made more explicit in the spec texts.

Reply:
SPARQL does not support `@prefix`. TriG does, of course, and any examples should be updated, but I don't think it necessary to highlight the use of `@prefix` or indeed `prefix` in the TriG grammar section given that it is explicitly derivative of Turtle. If it proves confusing, we can re-examine this.

`@prefix` is case sensitive, but `PREFIX` is not. Odd mixes of the `PREFIX` form are also acceptable, and I think we have tests for this. I think it's sufficient to say that this form is case-insensitive with an example using `prefix` and `base` to demonstrate.